integration-test/1955-chinese-parser.py

```diff
@@ -13,6 +13,125 @@ def _tile_centre(z, x, y):
+                u"max_areakm": 1748,
+                u"min_areami": 84,
+                u"max_areami": 675,
+                u"min_perkm": 126,
+                u"max_perkm": 755,
+                u"min_permi": 78,
+                u"max_permi": 469,
+                u"min_bbxmin": -122.516667,
+                u"max_bbxmin": -122.516667,
+                u"min_bbxmax": -122.358333,
+                u"max_bbxmax": -121.733333,
+                u"min_bbymin": 37.191667,
+                u"max_bbymin": 37.575,
+                u"min_bbymax": 37.816667,
+                u"max_bbymax": 38.041667,
+                u"mean_bbxc": -122.301354,
+                u"mean_bbyc": 37.622288,
+                u"timezone": u"America\/Los_Angeles",
+                u"un_fid": 570,
+                u"pop1950": 1855,
+                u"pop1955": 2021,
+                u"pop1960": 2200,
+                u"pop1965": 2361,
+                u"pop1970": 2529,
+                u"pop1975": 2590,
+                u"pop1980": 2656,
+                u"pop1985": 2805,
+                u"pop1990": 2961,
+                u"pop1995": 3095,
+                u"pop2000": 3236,
+                u"pop2005": 3387,
+                u"pop2010": 3450,
+                u"pop2015": 3544,
+                u"pop2020": 3684,
+                u"pop2025": 3803,
+                u"pop2050": 3898,
+                u"min_zoom": 2.7,
+                u"wikidataid": u"Q62",
+                u"wof_id": 85922583,
+                u"capalt": 0,
+                u"name_en": u"San Francisco",
+                u"name_de": u"San Francisco",
+                u"name_es": u"San Francisco",
+                u"name_fr": u"San Francisco",
+                u"name_pt": u"São Francisco",
+                u"name_ru": u"Cан-франциско",
+                u"name_zh": u"旧金山",
+                u"name_ar": u"سان فرانسيسكو",
+                u"name_bn": u"সান ফ্রান্সিস্কো",
+                u"name_el": u"σαν φρανσίσκο",
+                u"name_hi": u"सैन फ्रांसिस्को",
+                u"name_hu": u"San Francisco",
+                u"name_id": u"San Francisco",
+                u"name_it": u"San Francisco",
+                u"name_ja": u"サンフランシスコ",
+                u"name_ko": u"샌프란시스코",
+                u"name_nl": u"San Francisco",
+                u"name_pl": u"San Francisco",
+                u"name_sv": u"San Francisco",
+                u"name_tr": u"San Francisco",
+                u"name_vi": u"San Francisco",
+                u"ne_id": 1159151479,
+                u"name_fa": u"سان فرانسیسکو",
+                u"name_he": u"סן פרנסיסקו",
+                u"name_uk": u"Cан-франциско",
+                u"name_ur": u"سان فرانسسکو",
+                u"name_zht": u"舊金山"
+            })
+        )
+
+        # the min_zoom is 2.7 so it should appear at zoom 3
+        zoom = 3
+        x, y = deg2num(lat, lon, zoom)
+        self.assert_has_feature(
+            zoom, x, y, 'places', {
+                'id': 26819236,
```

Comment:
test that the source is coming from `naturealearthdata.com` and it's `kind: locality`.

Reply:
> test that the `source: naturealearthdata.com` and it's `kind: locality`.

done